tests/backtest/test_portfolio.py

```diff
@@ -141,7 +146,8 @@ def test_adjustment_success_sell(assets: dict, asset_pair: AssetPair) -> None:
 
 def test_adjustment_failure_sell(assets: dict, asset_pair: AssetPair) -> None:
```

Comment:
I think we should test the case where we fail explicitly due to transaction fees for both Buy and Sell, otherwise LGTM